mlflow/utils/databricks_utils.py

```diff
@@ -937,7 +937,7 @@ def get_config(self):
 
 def get_databricks_nfs_temp_dir():
     entry_point = _get_dbutils().entry_point
-    if getpass.getuser() == "ROOT":
+    if getpass.getuser() == "root":
```

Comment:
```suggestion
    if getpass.getuser().lower() == "root":
```

Reply:
Does `getuser` always return a string?

Reply:
I think so. according to doc https://github.com/databricks/universe/pull/595676

if it can't get valid user name, exception is raised.

and this is only invoked in databricks runtime, I have tested in databricks it is either `root` (assigned cluster) or `spark-xxxx` (shared cluster)